src/mlpack/core/tree/spill_tree/spill_tree.hpp

```diff
@@ -90,8 +90,8 @@ class SpillTree
   //! The number of points of the dataset contained in this node (and its
   //! children).
   size_t count;
-  //! The list of indexes of points contained in this node (non-null for
-  //! leaf nodes).
+  //! The list of indexes of points contained in this node (non-NULL if the node
```

Comment:
Interesting, "index" is one of those rare words that have two different plurals in English, didn't know that.

Reply:
Yeah, it's sometimes hard to know which one to use. :smile: